packages/core/test/router/router.test.ts

```
@@ -1,29 +1,29 @@
-import { beforeEach, it, vi } from 'vitest'
+import { beforeEach, it } from 'vitest'
 import { performHybridNavigation, router } from '../../src/router/router'
 import { getRouterContext, registerHook } from '../../src'
-import { fakeRouterContext, fakePayload, mockUrl } from '../utils'
-
-beforeEach(() => {
-	fakeRouterContext()
-	vi.stubGlobal('console', {
-		warn: vi.fn(),
```

These mocked functions were hiding an error that was being thrown during the request to the mocked server that said that there was no handler defined for the given request.

To define the handlers, you must either pass them in when calling `setupServer`, or you can call `resetHandlers()` and pass them in to an existing server.